build/buildControl.js

```diff
@@ -547,6 +547,15 @@ define([
 	bc.optimize = fixupOptimize(bc.optimize);
 	bc.layerOptimize = fixupOptimize(bc.layerOptimize);
 
+	if(/closure/.test(bc.optimize) || /closure/.test(bc.layerOptimize)){
+		bc.optimizeOptions = bc.optimizeOptions || {};
+		// ECMASCRIPT_2017 is necessary to avoid throwing errors on block-scoped functions
+		// https://github.com/google/closure-compiler/issues/3189
+		bc.optimizeOptions.languageIn = bc.optimizeOptions.languageIn || 'ECMASCRIPT_2017';
```

Comment:
Should this be ECMASCRIPT_2019 to match the stable version here?
https://github.com/google/closure-compiler/blob/v20200112/src/com/google/javascript/jscomp/CompilerOptions.java#L3217

Reply:
Another option would be to upgrade Closure to v20201006 and use ECMASCRIPT_2020: https://github.com/google/closure-compiler/wiki/Releases#october-6-2020-v20201006